### DISCUSSION TITLE
Using python-can for LIN Bus with Vector VN1611

### DISCUSSION THREAD

**Discussion by ReFlo** [hardbyte/python-can#1522] [Q&A]
Hello at all :-) 
I have a VN1611 interface that supports CAN and LIN. At the moment the case occurred that I want to use this interface for monitor the LIN bus communication. Additionally I would like to be master or slave in the BUS and receive and send messages.  In the documentations I see that LIN is somehow supported ( XL_BUS_TYPE_LIN ). My question is if the LIN bus is really supported. What functionalities are there and how do I set it up properly. 
I am new in this field and would appreciate some help.

Thank you !!

**Comment by zariiii9003** [comment]
Hello, python-can only supports CAN. However, the python-can VectorBus might be a good starting point to implement your own LIN class. Check the Vector XL Driver Library docs.

**Reply by ReFlo** [reply]
Thanks for your answer !

**Reply by filogold** [reply]
Have you found a way to communicate over LIN?

**Reply by ReFlo** [reply]
I used a different hardware and used their code 😄
